### PR TITLE
Post: openQA and PowerPC

### PR DESCRIPTION
**Pull request by grisu48**
Adds the blog post about openQA and PowerPC.

### CODE REVIEW

File: content/posts/2024/2024-11-11-openQA_and_PowerPC.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+author: phoenix
+type: post
+date: 2024-11-11T11:14:48+01:00
+categories:
+  - openqa
+tags:
+  - openqa
+  - PowerPC
+---
+Due to recent changes in the worker configuration, we needed to reconfigure some of the PowerPC jobs in openQA. This triggered a couple of questions regarding the availability of openQA worker, worker backends, their differences and their caveats. This blog post should act as a quickstart/overview guide for people getting into OpenQA testing on the PowerPC architecture.
```
Comment (okurz) [inline content/posts/2024/2024-11-11-openQA_and_PowerPC.md:12]: You might want to clarify that actually no changes were done in upstream openQA. You are talking about a SUSE-internal infrastructure change only.

**Review by okurz — CHANGES_REQUESTED** [review]

File: content/posts/2024/2024-11-11-openQA_and_PowerPC.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* `ppc64le_hmc` is more likely to run on more recent hardware (Power10), but comes with caveats (see [below](#caveats))
+* `ppc64le_hmc` is not (yet) available on [openqa.opensuse.org](https://openqa.opensuse.org)
+
+When selecting a PowerPC backend, use the [caveats section](#caveats) below for a help to decide which backend fits your use case better.
+
+# Different virtual machine modi
+
+On PowerPC everything is virtualized. The hardware is designed to be a hypervisor from ground up. Even in OPAL, it's most basic bare-metal configuration, the OS will run as the only VM on the internal hypervisor.
+
+The internal hypervisor is backed directly into a PowerPC and is managed by the VIOS (virtual input output server). VIOS is AIX based IO hub system which runs as a separate essential VM on the PowerPC machine itself. The customary way of managing virtual machines is to use either Novalink or HMC. HMC is an external management solution, typically running on a separate host and able to manage a cluster of PowerPC machines from one single host (similar to a Salt master). Novalink is used as internal management interface from the same host.
```
[the remark this answers is not in the document]
Comment (grisu48) [reply content/posts/2024/2024-11-11-openQA_and_PowerPC.md:30]: backed as in https://plainenglish.com/expressions/backed-by/, but I need to write it then as "backed by PowerPC directly". Thanks for the note!

**Review by okurz — APPROVED** [review]

**Comment by nicksinger** [comment]
LGTM :+1:

**Comment by grisu48** [comment]
Thank you!

**Pull request merged**